Slider/Assets/Scripts/Sliders/Grid/AreaSTiles/MountainSTile.cs

```diff
@@ -4,8 +4,67 @@
 
 public class MountainSTile : STile
 {
-    public MountainSTile()
+    public override void Init()
```

Comment:
try and call `base.Init();` instead of copying code if possible (haha DRY am i right)